<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorSetup</class>
 <widget class="QWidget" name="SensorSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Step Motor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBox_step_comport">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>COM1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM19</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM23</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM25</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_step_baudrate">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_step_com">
        <property name="text">
         <string>Open COM</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>EIC Laser</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QComboBox" name="comboBox_laser291_comport">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>COM1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM19</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM23</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM25</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Previous BaudRate&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_laser291_pre_baudrate">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500000</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Open BaudRate&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_laser291_baudrate">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500000</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_laser291_open">
        <property name="text">
         <string>Open COM</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QGroupBox" name="groupBox_right_dcmotor">
       <property name="title">
        <string>Right DC Motor</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QComboBox" name="comboBox_dcmotor_right_comport">
              <property name="currentIndex">
               <number>5</number>
              </property>
              <item>
               <property name="text">
                <string>COM1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM9</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM11</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM12</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM13</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM14</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM15</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM17</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM18</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM19</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM20</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM21</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM22</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM23</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM24</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM25</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_dcmotor_right_baudrate">
              <property name="currentIndex">
               <number>3</number>
              </property>
              <item>
               <property name="text">
                <string>9600</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>19200</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>38400</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>115200</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_dcmotor_right_comport">
            <property name="text">
             <string>Open COM</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_left_dcmotor">
       <property name="title">
        <string>Left DC Motor</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QComboBox" name="comboBox_dcmotor_left_comport">
              <property name="currentIndex">
               <number>4</number>
              </property>
              <item>
               <property name="text">
                <string>COM1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM9</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM11</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM12</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM13</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM14</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM15</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM17</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM18</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM19</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM20</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM21</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM22</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM23</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM24</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM25</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_dcmotor_left_baudrate">
              <property name="currentIndex">
               <number>3</number>
              </property>
              <item>
               <property name="text">
                <string>9600</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>19200</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>38400</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>115200</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_dcmotor_left_comport">
            <property name="text">
             <string>Open COM</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>SLAM Laser</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QComboBox" name="comboBox_laser291_comport_2">
        <property name="currentIndex">
         <number>6</number>
        </property>
        <item>
         <property name="text">
          <string>COM1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM19</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM23</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM25</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_laser291_pre_baudrate_2">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500000</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_laser291_baudrate_2">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500000</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_laser291_open_2">
        <property name="text">
         <string>Open COM</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
